### PR TITLE
Add rubygem-smart_proxy_ansible package [rpm]

### PR DESCRIPTION
**Pull request by iNecas**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

### CODE REVIEW

**Comment by dLobatog** [comment]
https://github.com/theforeman/foreman-packaging/pull/1336

**Comment by iNecas** [comment]
I'm updating this to reflect the changes we did as part of https://github.com/theforeman/foreman-packaging/pull/1321#issuecomment-248320382 the proxy-rex-ssh and this are very similar in a way they are being deployed

**Comment by iNecas** [comment]
Updated to match the approach in https://github.com/theforeman/foreman-packaging/pull/1321#issuecomment-248320382

File: rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Group: Applications/System
+License: GPLv3
+URL: https://github.com/theforeman/smart_proxy_ansible
+Source0: http://rubygems.org/downloads/%{gem_name}-%{version}.gem
+
+Requires: %{?rhel:tfm-}rubygem(smart_proxy_dynflow_core) >= 0.1.5
+Requires: %{?rhel:tfm-}rubygem(foreman_ansible_core)
+Requires: foreman-proxy >= 1.11.0
+Requires: rubygem(smart_proxy_dynflow) >= 0.1.0
+Requires: rubygem(smart_proxy_dynflow) < 0.2.0
```
Comment (domcleal) [inline rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec]: `< 1.0`, it's `~> 0.1`

**Comment by iNecas** [comment]
Yes, all foreman_ansible, foreman_ansible_core and smart_proxy_ansible are meant for 1.13

**Comment by domcleal** [comment]
foreman_ansible_core's now been built for 1.13, per the discussion in #1335 too.

**Comment by domcleal** [comment]
Merged and will build, thanks @iNecas.

**Pull request merged**